docs/root/configuration/overview/extension.rst

```diff
@@ -90,3 +92,20 @@ common subscription statistics, it also provides the following:
   config_reload, Counter, Total number of successful configuration updates
   config_fail, Counter, Total number of failed configuration updates
   config_conflict, Counter, Total number of conflicting applications of configuration updates; this may happen when a new listener cannot reuse a subscribed extension configuration due to an invalid type URL.
+
+
+Listener Filters
+
+For Listener filters, the discovery service configuration is: :ref:`dynamic listener filter
```

Comment:
Are we ready to remove `[#not-implemented-hide:]` 
 for the API?https://github.com/envoyproxy/envoy/blob/469f7bbd5951a29dfc24a3a05ca01459b835da2c/api/envoy/config/listener/v3/listener_components.proto#L360-L361

Reply:
Yeah, done!